presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -150,6 +150,12 @@ public static PlanMatchPattern constrainedIndexSource(String expectedTableName,
                 .addColumnReferences(expectedTableName, columnReferences);
     }
 
+    public static PlanMatchPattern constrainedTableScanWithTableLayout(String expectedTableName, Map<String, Domain> constraint, Map<String, String> columnReferences)
```

Comment:
put this method where the other table scan related method is.

Why not to put `TableLayoutHandleMatcher.any()` as an argument and then `WithTableLayout` suffix could be removed?